lib/ttfunk/table/head.rb

```diff
@@ -23,6 +23,12 @@ class Head < TTFunk::Table
       attr_reader :index_to_loc_format
       attr_reader :glyph_data_format
 
+      # Long date time (used in TTF headers) is defined here:
+      # https://developer.apple.com/fonts/TrueType-Reference-Manual/RM06/Chap6.html
+
+      # January 1, 1904 00:00:00 UTC offset from Epoch
+      LDT_BASIS_OFFSET = 2_082_844_800
```

Comment:
Why not use `Time.new(1904, 1, 1)` and use that for calculations?

Reply:
Sure, that'd be fine and probably clearer.